instrumentation/kafka/README.md

```diff
@@ -0,0 +1,26 @@
+# Brave Kafka instrumentation

```

Comment:
Might be good to make a note on the readme that this is only compatible with the recent versions of Kafka